applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -154,6 +154,52 @@ def Test(self, mu_test=None):
 
 
 
+    def RunFOM(self, mu_run=None):
+        if mu_run is None:
+            mu_run = ['single case with parameters already contained in the ProjectParameters.json and CustomSimulation']
```

Comment:
Why assigning a value to it? Wouldn't it be better to keep the `None` default value?

Reply:
I think it is a bit hacky. What if we default as `mu_run = [None]`?

Reply:
This one I like more. That would also solve the comment above